staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/types.go

```diff
@@ -220,6 +220,11 @@ type GroupResources struct {
 	// Any empty list implies every resource kind in the API group.
 	// +optional
 	Resources []string `json:"resources,omitempty" protobuf:"bytes,2,rep,name=resources"`
```

Comment:
Does the v1alpha1 Resources support the "resource/subresource" logic?
Do we need to update docstring for Resources?